src/style.css

```diff
@@ -1,3 +1,65 @@
-body {
+:root {
+  --blue-color: #00acdc; /* Определение переменной для синего цвета */
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example: `font-family: Roboto, Arial, sans-serif;`.